pl_bolts/datamodules/vision_datamodule.py

```diff
@@ -0,0 +1,131 @@
+import os
+from abc import abstractmethod
+from typing import List, Optional, Tuple, Union
+
+import torch
+from pytorch_lightning import LightningDataModule
+from torch.utils.data import DataLoader, Dataset, random_split
+
+
+class VisionDataModule(LightningDataModule):
+
+    EXTRA_ARGS = {}
+    #: Dataset class to use
+    DATASET_CLASS = ...
+    name: str = ""
+    #: A tuple describing the shape of the data
+    DIMS: tuple = ...
+
+    def __init__(
+        self,
+        data_dir: Optional[str] = None,
+        val_split: Union[int, float] = 0.2,
+        num_workers: int = 16,
+        normalize: bool = False,
+        seed: int = 42,
+        batch_size: int = 32,
+        *args,
+        **kwargs,
+    ):
```

Comment:
Recently in #388 by @briankosw, we added `pin_memory`, `shuffle` and `drop_last` to all datamodules, so could you include them here, too?